src/app/app.routes.ts

```diff
@@ -46,6 +46,15 @@ export const routes: Routes = [
       ),
     title: 'Download or Upload Datasets',
   },
+  {
+    path: 'profile',
+    canActivate: [() => inject(AuthService).guardAuthenticated()],
+    loadComponent: () =>
+      import('./portal/features/account/account.component').then(
+        (m) => m.AccountComponent,
+      ),
+    title: 'User Profile',
```

Comment:
```suggestion
    title: 'User Account',
```

We originally called it the "Profile" page, but then changed it to "Account" page, we should this wording now everywhere.